test/terraform_aws_s3_private_bucket_test.go

```diff
@@ -46,8 +75,108 @@ func AssertS3BucketEncryptionEnabledE(t *testing.T, region string, bucketName st
 	return nil
 }
 
-// An example of how to test the Terraform module in examples/terraform-aws-s3-example using Terratest.
-func TestTerraformAwsS3Example(t *testing.T) {
+func AssertS3BucketBlockPublicACLEnabled(t *testing.T, region string, bucketName string) {
```

Comment:
This is more informational, I'm now thinking it makes sense to eventually migrate these over to terratest so we can re-use them in other modules.